<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recover</class>
 <widget class="QDialog" name="Recover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create or Recover</string>
  </property>
   <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="text">
      <string>Let's create your wallet! Some info: A backup of the wallet will be created in the form of a list of words called a mnemonic. This list must never be given to anyone as it defines all of your keys, and thus any funds you have in this wallet.

The mnemonic scheme used is called BIP39. You can either create a new BIP39 wallet here, or recover an existing one by entering a valid BIP39 word list.

If you're not sure, click OK, and a secure wallet will be generated for you. You will be shown a word list on the next screen, please note down these words in a safe location.

(Advanced: if you'd rather not generate a wallet with the mnemonic feature, you can restart Zcoin-Qt now with the usemnemonic=0 option, either in your zcoin.conf file, or passed as an argument to the Zcoin-Qt program).
      </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
<item>
 <widget class="QFrame" name="createorrecover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>30</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QRadioButton" name="createNew">
     <property name="text">
      <string>Create new wallet</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="recoverExisting">
     <property name="text">
      <string>Recover existing wallet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
     <widget class="QFrame" name="numberofwords">
     <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QRadioButton" name="use12">
       <property name="text">
        <string>12 Words</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="use24">
       <property name="text">
        <string>24 Words</string>
       </property>
      </widget>
     </item>
     </layout>
    </widget>
     </item>
    </layout>
   </item>
 <item>
  <spacer name="horizontalSpacer">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="sizeType">
    <enum>QSizePolicy::Fixed</enum>
   </property>
   <property name="sizeHint" stdset="0">
    <size>
     <height>15</height>
    </size>
   </property>
  </spacer>
 </item>
 <item>
  <layout class="QHBoxLayout" name="horizontalLayout2">
   <item>
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Input words here:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
    <item>
     <widget class="QLineEdit" name="mnemonicWords"/>
    </item>
  </layout>
 </item>
    <item>
     <spacer name="horizontalSpacer1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
   <item>
   <widget class="QFrame" name="togglepassphrase">
    <layout class="QHBoxLayout" name="horizontalLayout5">
     <item>
      <widget class="QCheckBox" name="usePassphrase">
       <property name="text">
        <string>Use passphrase (Your mnemonic can be encrypted with an addional passphrase)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </item>
 <item>
   <layout class="QHBoxLayout" name="horizontalLayout3">
    <item>
     <widget class="QLabel" name="textLabel3">
      <property name="text">
       <string>Input mnemonic passphrase here:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mnemonicPassPhrase">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     </widget>
    </item>
   </layout>
 </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout4">
      <item>
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>Input mnemonic passphrase again:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mnemonicPassPhrase2">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="errorMessage">
      <property name="text">
       <string></string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Recover</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Recover</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>